<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>791</width>
      <height>561</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="streaming">
     <widget class="QSlider" name="Time">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>410</y>
        <width>511</width>
        <height>41</height>
       </rect>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="Menu">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/bmenu.png</normaloff>../resources/icons/bmenu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Prev">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>470</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/rewind.png</normaloff>../resources/icons/rewind.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="PausePlay">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>470</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/play.jpg</normaloff>../resources/icons/play.jpg</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="Next">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>470</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/ff.png</normaloff>../resources/icons/ff.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="AlbumArt">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>400</width>
        <height>400</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>110</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Title: </string>
      </property>
     </widget>
     <widget class="QLabel" name="Artist">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>190</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Artist:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Album">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>270</y>
        <width>281</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Album:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings1">
     <widget class="QPushButton" name="Menu_Exit">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="AlbumFullToggle">
      <property name="geometry">
       <rect>
        <x>81</x>
        <y>160</y>
        <width>191</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Album Full Screen</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="NetworkMenuButton">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>230</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Network Menu</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="AlbumArtwork">
     <widget class="QLabel" name="Big_Album">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>10</y>
        <width>550</width>
        <height>550</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../resources/blank.png</pixmap>
      </property>
     </widget>
     <widget class="QPushButton" name="BA_Exit">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>510</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Menu_2">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/bmenu.png</normaloff>../resources/icons/bmenu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Network">
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>190</y>
        <width>301</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>250</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Mask:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>300</y>
        <width>311</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Gateway</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>360</y>
        <width>321</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Wifi</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>50</y>
        <width>211</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Network</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Menu_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/bmenu.png</normaloff>../resources/icons/bmenu.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>32</width>
        <height>32</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="BA_Exit_2">
      <property name="geometry">
       <rect>
        <x>730</x>
        <y>510</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>exit</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
